api/types/database.go

```diff
@@ -832,6 +842,15 @@ func (d *DatabaseV3) CheckAndSetDefaults() error {
 		}
 	}
 
+	if oracle := d.GetOracle(); oracle != nil {
+		if oracle.AuditPullInterval < Duration(time.Second) {
```

Comment:
Isn't 1s too frequent? Should we even expose this as configuration parameter? What if we just default to 10s without letting users to change it?

Reply:
Agree that 1s if probably too frequent for prod env but I wanted to cover  testing scenario where you want to see the audit logs as soon as possible. 

I have aligned this logic to min 10s  as pull interval but I still think that just in case this value should be configurable. 
Does this sound resonable ?

Reply:
What scenario would a user want to change it?

Reply:
> I still think that audit_pull_interval knob is unnecessary (it's just unclear in which case users would need to tweak it) but I don't think it's a blocker so approving.

Removed the `audit_pull_interval` with 20s as default pull interval.